src/cascadia/TerminalSettingsModel/JsonUtils.h

```diff
@@ -89,7 +89,7 @@ namespace Microsoft::Terminal::Settings::Model::JsonUtils
     {
         static constexpr std::optional<T> EmptyV() { return std::nullopt; }
         static constexpr bool HasValue(const std::optional<T>& o) { return o.has_value(); }
-        // We can return a reference here because the original value is stored inside an std::optional
+        // We can return a reference here because the original value is stored inside a std::optional

```

Comment:
Huh, this is a weird one for its _pronunciation_ (ess). I'm cool with it.

Reply:
I surveyed some people (after the fact), the results vary, most people offered `standard` or variations like `stud`, a couple did offer `s t d` (I would avoid that because of the other meaning of S T D...), someone from a standards perspective weighed in with `stud` as it's shortest to speak.

Language is such an amazing thing.